assets/src/tinymce/web-stories.js

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
This too should be it's own package. It will make managing dependientes much easier.

Reply:
Fixed.